src/Control/Lens/Indexed.hs

```diff
@@ -82,43 +77,32 @@ module Control.Lens.Indexed
 
 import Prelude ()
 
-import Control.Applicative.Backwards
-import Control.Comonad.Cofree
-import Control.Comonad.Trans.Traced
-import Control.Monad (void, liftM)
-import Control.Monad.Trans.Identity
-import Control.Monad.Trans.Reader
-import Control.Monad.Trans.State.Lazy as Lazy
-import Control.Monad.Free
+import Data.Functor.WithIndex
+import Data.Functor.WithIndex.Instances ()
```

Comment:
Can you include a comment saying which instances in particular we are importing here?

Reply:
Added comment and moved to internal prelude to infect as many modules with orphans as possible.